core/include/scipp/core/string.h

```diff
@@ -52,13 +54,13 @@ std::string to_string(const MutableView<T, U> &mutableView) {
 }
 
 template <class T>
-std::string array_to_string(const T &arr, const std::string &unit = "");
+std::string array_to_string(const T &arr, const units::Unit &unit = units::ns);
```

Comment:
This default value does not make sense? Use `std::optional<units::Unit>` instead so you can pass an nullopt as default?

Reply:
OK. Makes sense.